frontend/src/app/[locale]/print/form/[applicationId]/[appFormId]/page.tsx

```diff
@@ -0,0 +1,65 @@
+import { Metadata } from "next";
```

Comment:
to make this more RESTful, consider putting it at `application/[applicationId]/form/[appFormId]/print`, or if we need a different root path, maybe `print/application/[applicationId]/form/[appFormId]`